src/lib/uTop_styles.ml

```diff
@@ -98,87 +98,107 @@ let load () =
         Lwt_log.error_f "cannot load styles from %S: %s: %s" fn func (Unix.error_message error)
     | exn -> Lwt.fail exn)
 
-let rec stylise_filter_layout stylise tokens =
-  match tokens with
+let stylise_filter_layout stylise tokens =
```

Comment:
We can probably make this code clearer by using something like `List.filter_map`